CONTRIBUTING.md

```diff
@@ -0,0 +1,144 @@
+
+- [Contribution Process](#contribution-process)
+- [Coding Rules](#coding-rules)
+- [Commit Message Guidelines](#commit-message-guidelines)
+- [Local Development Cheatsheet](#local-development)
+
+## <a name="contribution-process"></a> Contribution Process
+
+### File an issue
+
+Before you start any work, file an issue. If an issue already exists, comment on it and mention the `@blackbaud/skyux-maintainers` team so that we know you intend to work on it. The sooner we discuss potential code changes, the better.
+
+Keep in mind that the Blackbaud design council must approve changes that impact the UI, such as new components or changes to existing components. Contributions that do not have design approval must include detailed design specifications. The design approval process can take significant time if components are complex or designs require significant changes, so account for that in your development timeline. If you need help with the design approval process, ask the SKY UX team.
+
+### Discuss the contribution with the SKY UX team
+
+After you file an issue, discuss the architecture and implementation details, requirements, and timelines with the `@blackbaud/skyux-maintainers`. We will assign a developer to your issue to help answer questions.
+
+### Developer setup
+
+1. Fork the repo into your own GitHub repo.
+1. Create a branch off of `main`. Name it after the feature to contribute, and use kebab-case (e.g. `my-new-feature`).
+1. Clone your repo locally and run `nvm use` to ensure you have the preferred version of Node.js installed. Then run `npm install` to install all required dependencies. For more information on prerequisites, see [SKY UX prerequisites](https://developer.blackbaud.com/skyux/learn/get-started/prereqs).
+
+### Request a preliminary review
+
+Create a pull request with the initial implementation, functionality, and styling, and mention the `@blackbaud/skyux-maintainers` to request a review. Make changes and request additional reviews as necessary. For [large changes that effect more than 50 files](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#pull-requests), submit changes in multiple small pull requests instead of one large pull request.
+
+Wait until after the initial review to add tests and documentation because implementation details may change.
+
+### Add tests
+
+Write tests for your feature. All code requires 100 percent [unit test](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#unit-tests) code coverage, and components also require [visual regression tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#visual-regression-tests) and [accessibility tests](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#accessibility-tests). In addition, create [test fixtures](https://developer.blackbaud.com/skyux/contribute/contribution-process/guidelines#test-fixtures) for any new components.
```

Comment:
Is the part about visual regression tests correct? We removed that section of the contribution guidelines from the v5 docs as part of Braden's review and cleanup of the contribution guidelines, and I just realized that we still have this broken link in the contribution process tutorial. If visual regression tests are still necessary, then our v5 docs are wrong now and we need to add that content back to the contribution guidelines or that link will not work after we flip the switch to make v5 the primary site. If they are not required, we need to remove the link here, and I need to remove the soon-to-be-busted link from the contribution process tutorial.

Reply:
Good question. We're working on re-adding visual tests to our monorepo, so we can either keep that section in the docs, or remove it now (and re-add it later).